ckanext/unaids/theme/public/custom.css

```diff
@@ -216,3 +216,24 @@ section.resources H3{
 footer{
   flex: 1;
 }
+
+.vertical-align-top{
+  vertical-align: top !important;
```

Comment:
Not really as i'm placing this deep inside hyperlinks sitting inside list groups, so there's a lot of css we want to override. Usually we would use an inline `style=""` however I really wanted to avoid that, so this is one step in a better direction that having inline styles.